template/index.html

```diff
@@ -4,6 +4,24 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Primary Meta Tags -->
+<title>Portfolio Template - Open source</title>
+<meta name="title" content="Portfolio Template - Open source">
+<meta name="description" content="Build or customize your portfolio website using this super easy template. Free to use if you are a member of the community or a contributor to this project">
+
+<!-- Open Graph / Facebook -->
+<meta property="og:type" content="website">
+<meta property="og:url" content="https://cpro-portfolio-html.netlify.app/">
```

Comment:
Thanks @vermilion4 can you please replace the link with this github link to the project.

https://github.com/CommunityPro/portfolio-html

Reply:
no problem